components/scream/src/control/atmosphere_driver.cpp

```diff
@@ -327,6 +330,48 @@ void AtmosphereDriver::set_precipitation_fields_to_zero ()
   }
 }
 
+void AtmosphereDriver::setup_conservation_checks ()
```

Comment:
This routine looks like it always runs, regardless of a user option to run or not run the conservation checks. I'd like this routine to exit immediately if checks are not requested.

Separately, I think these conservation checks should have a different input control variable than enable_postcondition_checks, e.g., enable_column_conservation_checks.